### PR TITLE
add round active boolean to turn off the counter

### PR DESCRIPTION
**Pull request by octavioamu**
##### Description

- add "active" field on json data 
- add condition to disable the countdown display based on "active"
- set default dates to 14 days round

### CODE REVIEW

File: app/grants/views.py [hunk 1]
```diff
@@ -1147,7 +1150,7 @@ def grants_by_grant_clr(request, clr_round):
         if _type.get("keyword") == grant_type:
             grant_label = _type.get("label")
 
-    clr_round, round_start_date, round_end_date = get_clr_rounds_metadata()
+    clr_round, round_start_date, round_end_date, round_active = get_clr_rounds_metadata()
```
Comment (thelostone-mc) [inline app/grants/views.py:1153]: `clr_round, round_start_date, round_end_date, _ = get_clr_rounds_metadata()`

File: app/grants/views.py [hunk 2]
```diff
@@ -970,7 +973,7 @@ def grants_by_grant_type(request, grant_type):
...
```
Comment (thelostone-mc) [inline app/grants/views.py:976]: `clr_round, round_start_date, round_end_date, _ = get_clr_rounds_metadata()`

**Comment by thelostone-mc** [comment]
Can we rename `active` to `show_banner` or a more meaningful name ?

File: app/grants/views.py (hunk 2 above)
Comment (thelostone-mc) [reply app/grants/views.py]: fixed this

File: app/grants/views.py (hunk 1 above)
Comment (thelostone-mc) [reply app/grants/views.py]: fixed this

**Comment by octavioamu** [comment]
> Can we rename `active` to `show_banner` or a more meaningful name ?

I don't think so, this doesn't hide the banner just the counter, we don't want to hide the banner. 
Also the idea is just follow the state of the round /active or not  without worry where will be used.

File: app/grants/views.py [hunk 3]
```diff
@@ -973,7 +973,7 @@ def grants_by_grant_type(request, grant_type):
         if _type.get("keyword") == grant_type:
             grant_label = _type.get("label")
 
-    clr_round, round_start_date, round_end_date, round_active = get_clr_rounds_metadata()
+    _, round_start_date, round_end_date, _ = get_clr_rounds_metadata()
```
Comment (octavioamu) [inline app/grants/views.py:976]: this conflict with `from django.utils.translation import gettext_lazy as _` why need to be _ ?

Comment (gdixon) [reply app/grants/views.py:976]: Yeah good point, we can import that as anything 👍 Theres 124 uses in `app/dashboard/views.py` - should we go for something like `__` - Or do we want to change it to something more meaningful?

Comment (octavioamu) [reply app/grants/views.py:976]: My opinion is keep the name of what that function return, is more easy to understand and learn about the code.

File: app/grants/views.py [hunk 4]
```diff
@@ -102,6 +102,7 @@ def get_clr_rounds_metadata():
         clr_round = CLR_ROUND_DATA['round_num']
         start_date = CLR_ROUND_DATA['round_start']
         end_date = CLR_ROUND_DATA['round_end']
+        show_bannner = CLR_ROUND_DATA['show_bannner']
```
Comment (gdixon) [inline app/grants/views.py:105]: I think `clr_active`/`active` makes more sense - we can use this to toggle the round dates in the trust requirements banner too 👍

**Pull request merged**